### DISCUSSION TITLE
Can't used scoped packages

### DISCUSSION THREAD

**Discussion by laurenskling** [strapi/strapi#6742] [Legacy Topics]
https://github.com/strapi/strapi/blob/9ed11a66829dbdafabb49cb8813dd4d891a247de/packages/strapi-plugin-email/config/functions/bootstrap.js#L9

This line makes it impossible to install my namespaced package: https://www.npmjs.com/package/@laurenskling/strapi-provider-email-postmark

**Comment by derrickmehaffy** [comment]
Moving to a discussion

**Comment by derrickmehaffy** [comment]
@soupette can we dig a little deeper on how packages are called up? I assume Strapi is just recursively searching through the top level of node_modules looking for `strapi-*` ?

**Reply by tudorilisoi** [reply]
@laurenskling 
there is a workaround in the docs https://github.com/strapi/strapi/pull/6753
